### PR TITLE
fixes #8442 - Associate user model to Audit

### PR DESCRIPTION
**Pull request by shlomizadok**
This will add a user_id to the audit from now on. 
Shall I also add a migration to add user_id to all previous audits?

### CODE REVIEW

**Comment by ohadlevy** [comment]
@orrabin can you review please?
@shlomizadok can you add a couple of tests for association of users and audits?

**Comment by orrabin** [comment]
@shlomizadok LGTM
@ohadlevy audits have very few tests today, maybe we should add a different issue about testing audits and merge this PR to fix the error quickly?

File: app/models/concerns/audit_extensions.rb [hunk 1]
```diff
@@ -36,7 +37,7 @@ module AuditExtensions
   private
 
   def ensure_username
-    self.username ||= User.current.to_s rescue ""
+    self.username ||= User.current.to_s rescue ''
```
Comment (ohadlevy) [inline app/models/concerns/audit_extensions.rb]: rescue nil?

**Comment by isratrade** [comment]
Thanks @shlomizadok, merged as 411ebf4108d2d01d8651b659fb9fd07e35996735. I made the mistake again, when editing the commit message, and forgot to change the owner back to you. Credit goes to @shlomizadok

**Pull request closed without merging**